testing/simapp/simd/cmd/testnet.go

```diff
@@ -93,7 +93,7 @@ Example:
 	return cmd
 }
 
-const nodeDirPerm = 0755
+const nodeDirPerm = 0o755
```

Comment:
what does this change do? I'd prefer to stick with what the [sdk does](https://github.com/cosmos/cosmos-sdk/blob/624539a9f37271ad6a27014db5e76fa3f5b81a3a/simapp/simd/cmd/testnet.go#L193)

Reply:
sdk now does this.

That change is codument in golang... 1 moment

Reply:
https://github.com/golang/go/issues/33363

Reply:
Indeed SDK [does this also now](https://github.com/cosmos/cosmos-sdk/blob/main/simapp/simd/cmd/testnet.go#L190). [Thanks as well](https://github.com/cosmos/cosmos-sdk/pull/11839) to @faddat. :)